xplique/metrics/stability.py

```diff
@@ -60,8 +61,8 @@ def __init__(self,
             raise ValueError(f"{distance} is not a valid distance.")
 
         # prepare the noisy masks that will be used to generate the neighbors
-        nb_variables = np.prod(inputs.shape[1:])
-        self.noisy_masks = tf.random.uniform((nb_samples, *inputs.shape[1:]), 0, 1.0 / nb_variables)
+        self.noisy_masks = tf.random.uniform((nb_samples, *inputs.shape[1:]), 0,
+                                             self.radius)
```

Comment:
I think that removing the division by nb_variables does not lead to the expected behavior. The noise norm esperance should be the radius, here it is nb_features\*radius.